working-open.md

```diff
@@ -51,3 +53,10 @@ Hypha operates with democratic organizing principles informed by our [values](./
 - Chat transcripts
 - Notes involving external parties and collaborators
 - Materials from project retrospectives
+
+**Sensitive and confidential**:
+
+- Passwords, credentials, and sensitive personal information
+- Data sets, software, and other content in project work that we don't have consent to share
+- Financial or other information of our clients and partners that we don't have consent to share
```

Comment:
```suggestion
- Financial information of our clients and partners
```
not sure what "other information" is here, if not covered by above line?

Reply:
Flagging term partner here too needs to be updated

Reply:
other information -> anything that we do not have consent to share, whether or not of financial or personal nature

```suggestion
- Financial or other information of our clients and collaborators that we don't have consent to share
```

Reply:
What would that be except "other content in project work" or the things identified above? "other business information"

Reply:
```suggestion
- Financial or other business information of our clients and collaborators that we don't have consent to share
```